app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -11,27 +10,49 @@ def self.call(*args)
       end
 
       def call
-        return if commented_on_welcome_thread? || received_notification?
+        return unless user
```

Comment:
I don't think that there's anything wrong with this check here per se...but I'm curious: if we're expecting a rake task to call this generator, would we ever try to call this without a user? Should we instead modify line 5 to `find_by!` and `raise` when a user isn't found, instead?

Reply:
That sound a lot better. I'll do that